README.md

```diff
@@ -152,13 +152,21 @@ brew install pastel
 
 Check out the [release page](https://github.com/sharkdp/pastel/releases) for binary builds.
 
-### Via cargo
+### Via cargo (source)
+
+If you do not have cargo, install using [rust's installation documentation](https://doc.rust-lang.org/book/ch01-01-installation.html).
 
 If you have Rust 1.35 or higher, you can install `pastel` from source via `cargo`:
 ```
 cargo install pastel
 ```
 
+If you have Rust 1.35 or higher, you can install `pastel` from source via `cargo` using this repository:
```

Comment:
I don't think we have to repeat "if you have Rust 1.35 or higher" here. Also, `cargo install` installs "from source" as well, so I would suggest to rephrase this to:

*Alternatively, you can install `pastel` from this repository using*

What do you think?

Reply:
Sure.  Sounds good.

I deleted the source repository so I cannot make the change.